library/formatting.inc.php

```diff
@@ -198,6 +173,7 @@ function DateFormatRead($mode = 'legacy')
 
 function DateToYYYYMMDD($DateValue)
 {
+    return DateFormatterUtils::DateToYYYYMMDD($DateValue);
```

Comment:
shouldn't all the code below this line now be removed?

Reply:
Removed this code.